### DISCUSSION TITLE
Walkthrough Video Stellar Soroban Quest Live 3 - Reverse Engineer

### DISCUSSION THREAD

**Discussion by chandrapratamar** [stellar/sorobanathon#57] [General]
I Created a Walkthrough Video for Stellar Soroban Quest Live 3 - Reverse Engineer
here's the link : https://youtu.be/VYg6MlHbXeI

**Comment by achallagundla** [comment]
Heya @darkvallen  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

**Reply by chandrapratamar** [reply]
Hi, i already filled the form,and still didnt receive the reward,is there any issue of my kyc or something?

**Reply by achallagundla** [reply]
Hey hey,
I have been OOO last week so hang tight as we process KYCs and get the awards out soon :)

**Reply by chandrapratamar** [reply]
Glad to hear that, Thanks for the information @achallagundla,  sorry to bother you,i am just affraid if there's something wrong with my form or kyc
